org.sofproject.ui/META-INF/MANIFEST.MF

```diff
@@ -25,4 +25,5 @@ Export-Package: org.sofproject.ui.editor,
  org.sofproject.ui.properties,
  org.sofproject.ui.wizards
 Bundle-Vendor: Intel Corporation
-Import-Package: com.google.common.collect;version="21.0.0"
+Import-Package: com.google.common.collect;version="21.0.0",
+ org.sofproject.gst.topo.ui.preferences
```

Comment:
Do not create a dependency on an optional plug-in for specific topology in the common base. Those preferences should be implemented in org.sofproject.ui, not in gst.topo if supposed to be used in the common login dialogs.

Reply:
Done

Reply:
Done